content/foundations/lifecycle.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Component lifecycle
+---
+
+These milestones summarize the maturity lifecycle of UI components in Primer. Components are required to meet all criteria _before_ being proceeding to a given milestone.
```

Comment:
```suggestion
These milestones summarize the maturity lifecycle of UI components in Primer. Components must meet all criteria _before_ proceeding to a given milestone.
```

Reply:
IMO "Components must meet all criteria _before_ proceeding to a given milestone" isn't super clear. 

Is there a concise way to say: "Components must meet all criteria **of a milestone** _before_ proceeding to **that milestone**"?

Reply:
I'm also a little on the fence about calling these "milestones" since we use that term a lot of other contexts (e.g. we have a "milestones" feature in GitHub). What if we called them "stages"?